docs/specs/aerogear-security-otp.asciidoc

```diff
@@ -1,5 +1,6 @@
 ---
-layout: basic
+layout: post
+section: docs
 title: AeroGear Security OTP Specification
```

Comment:
This spec is linked from `docs/guides/AeroGear-OTP.asciidoc`, I believe we should also make it available from Docs index.